modules/steam/autostart.nix

```diff
@@ -116,17 +116,15 @@ in
         enable = true;
         settings = {
           default_session = {
-            user = "jovian-greeter";
+            user = "root";
             command = "${pkgs.jovian-greeter}/bin/jovian-greeter ${cfg.user}";
           };
         };
       };
 
-      users.users.jovian-greeter = {
-        isSystemUser = true;
-        group = "jovian-greeter";
-      };
-      users.groups.jovian-greeter = {};
+      # We handle this ourselves in the greeter
+      systemd.services.plymouth-quit.enable = false;
+      systemd.services.plymouth-quit-wait.enable = false;
```

Comment:
You shouldn't disable `plymouth-quit-wait.service`. That's there so that other things know when plymouth actually quit.